components/automate-chef-io/Makefile

```diff
@@ -17,8 +17,19 @@ clean_swagger_files:
 	rm -rf ./$(SWAGGER_DIR)/*
 
 sync_swagger_files: clean_swagger_files
-	pushd ../../api/external && find . -name '*.swagger.json' | cpio -p -dumv ../../components/automate-chef-io/$(SWAGGER_DIR) && popd
-	pushd ../automate-gateway/api/ && find . -name '*.swagger.json' | cpio -p -dumv ../../automate-chef-io/$(SWAGGER_DIR) && popd
+	pushd ../../api/external && \
+		find . -name '*.swagger.json' | sed 's/^\.\///' | while read i ; do \
+			mkdir -p ../../components/automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` && \
+			cp $$i ../../components/automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` \
+		; done && \
+	popd
+
+	pushd ../automate-gateway/api/ && \
+		find . -name '*.swagger.json' | sed 's/^\.\///' | while read i ; do \
+			mkdir -p ../../automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` && \
+			cp $$i ../../automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` \
+		; done && \
+	popd
```

Comment:
If anyone would like to attempt to golf this down, `cd components/automate-chef-io` and `make sync_swagger_files`, then `git status` should show no changes to the swagger files (they are deleted and recopied on each invocation). Keep in mind this needs to run in a restricted env so things like `cpio` and `rsync` are probably not available.

Reply:
😃 _+1, insightful_